src/coreComponents/virtualElement/VirtualElementBase.hpp

```diff
@@ -35,7 +35,7 @@ class VirtualElementBase
 public:
 
   /// Default constructor.
-  VirtualElementBase() = default;
+  VirtualElementBase();
```

Comment:
This seems to have been removed by accident?

Reply:
nope....linking issue.

Reply:
But now that this constructor is not defined anywhere at all... there is no linking issue? I'm surprised. Anyway, it looks like `VirtualElement` objects aren't instantiated ever, so I would prefer to see a `= delete` default constructor rather than an undefined one.

Reply:
Well...It would have helped if I added the source file. But the deletion is a better idea.

Reply:
deleted